PriceQuoteIosTests/Salesman/SalesmanApiDataSourceTests.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SalesmanApiDataSourceTests.swift
+//  PriceQuoteIosTests
+//
+//  Created by Livia Vasconcelos on 27/09/20.
+//  Copyright © 2020 Livia Vasconcelos. All rights reserved.
+//
+
+import XCTest
+@testable import PriceQuoteIos
+
+class SalesmanApiDataSourceTests: XCTestCase {
+
+    func test_create_withCorrectFiels_requestSucceedes() {
+        let sut = SalesmanApiDataSourceImplMock()
+        let salesman = sut.salesman
+        let e = expectation(description: "Alamofire")
+        sut.status = .success
+        sut.create(salesman: salesman) { result in
```

Comment:
This is very cluttered, I would move this at least to a `setUp` block since it looks like setup

Reply:
Refactored!